app/models/donation.rb

```diff
@@ -0,0 +1,69 @@
+require 'pagarme'
+
+class Donation < ActiveRecord::Base
+  belongs_to :widget
+  has_one :mobilization, through: :widget
+  has_one :organization, through: :mobilization
+
+  after_create :capture_transaction
+  after_create :send_mail
+
+  def find_transaction
+    @transaction = PagarMe::Transaction.find_by_id(self.token)
+  end
+
+  def capture_transaction
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Assignment Branch Condition size for capture_transaction is too high. [20.12/15]](https://www.codacy.com/app/jlcfigueira/hub-api/file/2456391393/issues/source?bid=3141163&fileBranchId=3250265#l15)